live-build/config/hooks/vm-artifacts/83-etc-resolv-conf.binary

```diff
@@ -31,4 +31,4 @@
 # environment.
 #
 chroot binary rm -f /etc/resolv.conf
-chroot binary ln -s ../run/systemd/resolve/resolv.conf /etc/resolv.conf
+chroot binary ln -s ../run/systemd/resolve/stub-resolv.conf /etc/resolv.conf
```

Comment:
It might be nice to have a comment that indicates why `stub-resolv.conf` should be used.